src/redis-cli.c

```diff
@@ -3593,6 +3593,8 @@ static int evalMode(int argc, char **argv) {
         /* Call it */
         int eval_ldb = config.eval_ldb; /* Save it, may be reverted. */
         retval = issueCommand(argc+3-got_comma, argv2);
+        for (j = 0; i < argc+3-got_comma; i++) sdsfree(argv2[i]);
```

Comment:
please double check it, thanks.
```suggestion
        for (j = 0; j < argc+3-got_comma; j++) sdsfree(argv2[i]);
```